src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -55,15 +60,20 @@ var EventInterface = {
  * @param {DOMEventTarget} nativeEventTarget Target node.
  */
 function SyntheticEvent(dispatchConfig, targetInst, nativeEvent, nativeEventTarget) {
+  for (var i = 0; i < otherNullableProps.length; i++) {
+    delete this[otherNullableProps[i]];
+  }
   this.dispatchConfig = dispatchConfig;
   this._targetInst = targetInst;
   this.nativeEvent = nativeEvent;
+  this._destructored = false;
 
   var Interface = this.constructor.Interface;
   for (var propName in Interface) {
     if (!Interface.hasOwnProperty(propName)) {
       continue;
     }
+    delete this[propName];
```

Comment:
Should we be concerned about performance implications of `delete`ing the properties off the instance?